test/routes/src/server/routes/t_github.js

```diff
@@ -34,24 +40,26 @@ describe('The GitHub API endpoint', () => {
         nock.cleanAll();
       });
 
-      it('should successfully return', (done) => {
-        supertest(app)
-          .get('/github/snapcraft-yaml/anowner/aname')
-          .expect(200, done);
+      it('should satisfy all request mocks', async () => {
```

Comment:
Good point. I didn't want to repeat them in all tests, and separate test case sounded bit artificial.

Moving to `afterEach` sounds as a good compromise. But in the end it will probably also not really tell which test case made it fail.